integration/query_frontend_test.go

```diff
@@ -207,16 +216,14 @@ func TestQueryFrontendWithQueryResultPayloadFormats(t *testing.T) {
 					flags = mergeFlags(
 						BlocksStorageFlags(),
 						BlocksStorageS3Flags(),
-						map[string]string{
-							"-query-frontend.query-result-response-format": format,
-						},
 					)
 
 					minio := e2edb.NewMinio(9000, flags["-blocks-storage.s3.bucket-name"])
 					require.NoError(t, s.StartAndWaitReady(minio))
 
 					return "", flags
 				},
+				withHistograms: format == "protobuf",
```

Comment:
It seems to me that now we only test `protobuf+histograms`, or `json+floats`, but not `protobuf+floats` combination.

Reply:
hmm, good catch

Reply:
fixed by adding loops on the format as well